### ISSUE TITLE
Developers commands menu is nonresponsive

### ISSUE THREAD

**Issue by Nackoo2000** [breez/misty-breez#279]
When entering the developers screen the commands menu is not working.
Also, I can't manually enter a command in the field.

The top menu (3 dots) is working as it should

**Comment by erdemyerebasmaz** [comment]
Non-responsiveness is by choice. CLI on Developers is disabled until we have an `execute_command` API endpoint on the SDK, which is not planned in the near future.

We are likely to remove it before release.

**Comment by roeierez** [comment]
If the console is not usable (and not needed) then let's remove it @erdemyerebasmaz .
